frontend/dashboard/pages/PageLayout/DashboardHeader/SmallHeaderMenu/SmallHeaderMenuItem/SmallHeaderMenuItem.tsx

```diff
@@ -38,17 +36,17 @@ export const SmallHeaderMenuItem = ({
     );
   }
 
-  const linkItemClassName: string =
-    UrlUtils.extractLastRouterParam(menuItem.action.href) === currentRoutePath
-      ? classes.active
-      : '';
-
```

Comment:
Removing this as it does not make any sense to keep a link with `active` class when it opens in new tab